app/models/fuel/post.rb

```diff
@@ -57,11 +64,27 @@ def teaser_content
       teaser.present? ? teaser : content
     end
 
-    def to_html
-      markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML,
-          :autolink => true, :space_after_headers => true)
+    def to_html(content = raw_content)
+      markdown = Redcarpet::Markdown.new(Fuel::Html,
+          :autolink => true, :space_after_headers => true, fenced_code_blocks: true, disable_indented_code_blocks: true)
```

Comment:
Line is too long. [120/80]<br>Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.